AlgebraInLean/World0LeanIntroSOLUTIONS/Sheet3Solutions.lean

```diff
@@ -0,0 +1,143 @@
+/-
+This is a solutions sheet.
+-/
+
+import Mathlib.Tactic
+
+/-Credit for some exercises goes to Kevin Buzzard and his Formalizing Mathematics course OR to the
+Natural Number Game. If you would like to learn more about Lean, Buzzard's course goes more in depth
+in relation to numerous undergraduate math topics. When exercises are from either of these sources,
+they will be clearly marked so as to give credit.
+
+Formalising Mathematics can be found here:
+<https://www.ma.imperial.ac.uk/~buzzard/xena/formalising-mathematics-2024/index.html>-/
+
+/-In the Natural Number Game, you used the tactics:
+
+left
+right
+
+Let's do a couple warm up exercises from Formalizing Mathematics using these. Note that the ∨ symbol
```

Comment:
A quick reminder of what these two tactics do would be helpful, for someone who didn't finish NNG or simply forgot about `left` and `right`.